R/input.R

```diff
@@ -146,11 +146,11 @@ numericInput <- function(inputId, label, value, min = NA, max = NA, step = NA, w
 #' Change numeric input value and settings
 #'
 #' @param session The session object passed to function given to shinyServer.
-#' @param inputId The id of the input object.
+#' @param input_id The id of the input object.
 #' @param label The label to set for the input object.
 #' @param value The value to set for the input object.
 #' @param min Minimum value.
 #' @param max Maximum value.
 #' @param step Step size.
 #' @export
-updateNumericInput <- shiny::updateNumericInput
+update_numeric_input <- shiny::updateNumericInput
```

Comment:
And this should be rewritten with usage of out parameters used in uinumericinput i.e. input_id.

Reply:
More to that pay attention that our numericInput has `...` parameter that allows to customize this with more shiny.semantic parameters so the update could allow this as well in the future.